xbmc/video/VideoDatabase.cpp

```diff
@@ -2097,15 +2097,19 @@ int CVideoDatabase::SetDetailsForMovie(const std::string& strFilenameAndPath, co
     }
     // update our movie table (we know it was added already above)
     // and insert the new row
-    std::string sql = "update movie set " + GetValueString(details, VIDEODB_ID_MIN, VIDEODB_ID_MAX, DbMovieOffsets);
+    std::string sql = "UPDATE movie SET " + GetValueString(details, VIDEODB_ID_MIN, VIDEODB_ID_MAX, DbMovieOffsets);
     if (idSet > 0)
```

Comment:
I tried to do these four lines in a seperate method and just reuse it to construct the sql, but couldn't make it work :/